### ISSUE TITLE
Alias not working in 2.x

### ISSUE THREAD

**Issue by christobill** [mpollmeier/gremlin-scala#122]
After trying to replace -> (reserved by scala) by `⇒` or startPipe

```
[error] /home/christophe/today/shed/src/main/scala/com/misterbell/shed/api/engine/GraphQueryPipeline.scala:319: type mismatch;
[error]  found   : Seq[com.tinkerpop.gremlin.scala.GremlinScalaPipeline[Nothing,Nothing]]
[error]  required: Seq[com.tinkerpop.pipes.Pipe[T, ?]]
[error]     pipeline.or(posibleValues:_*)
[error]                 ^
[error] /home/christophe/today/shed/src/main/scala/com/misterbell/shed/api/engine/GraphQueryPipeline.scala:311: type mismatch;
[error]  found   : com.tinkerpop.gremlin.scala.GremlinScalaPipeline[Nothing,Nothing]
[error]  required: com.tinkerpop.gremlin.scala.GremlinScalaPipeline[T,T]
[error] Note: Nothing <: T, but class GremlinScalaPipeline is invariant in type S.
[error] You may wish to define S as +S instead. (SLS 4.5)
[error] Note: Nothing <: T, but class GremlinScalaPipeline is invariant in type E.
[error] You may wish to define E as +E instead. (SLS 4.5)
[error]       val orStep = step.or.map{s => applyStep(startPipe,s,excludeDeleted)}
[error]                                               ^
[info] com.tinkerpop.gremlin.scala.GremlinScalaPipeline[Nothing,Nothing] <: com.tinkerpop.gremlin.scala.GremlinScalaPipeline[T,T]?
[info] false
[error] /home/christophe/today/shed/src/main/scala/com/misterbell/shed/api/engine/GraphQueryPipeline.scala:312: type mismatch;
[error]  found   : Seq[com.tinkerpop.gremlin.scala.GremlinScalaPipeline[Nothing,Nothing]]
[error]  required: Seq[com.tinkerpop.pipes.Pipe[T, ?]]
[error]       pipeline.or(orStep: _*)
[error]   
```

Type is not inferred correctly.

```
  /** Start a new pipeline */
  def ->[S]: GremlinScalaPipeline[S, S] = new GremlinScalaPipeline[S, S]()
  def `⇒`[S] = ->
  def startPipe[S] = ->
```

should be replaced by 

```
/** Start a new pipeline */
  def ->[S]: GremlinScalaPipeline[S, S] = new GremlinScalaPipeline[S, S]()
  def `⇒`[S] = ->[S]
  def startPipe[S] = ->[S]
```

**Comment by joan38** [comment]
Great one.
Thanks for the PR.

**Comment by christobill** [comment]
I am using version 2.6.1in production
Would it be possible to tag a version 2.6.2 for this bug? 
I would be very grateful

**Comment by joan38** [comment]
As I said, I don't think the version 2.x is still supported/released anymore tho.
@mpollmeier Can confirm that.

**Comment by mpollmeier** [comment]
thank you! 
I have just released 2.6.2, should be in maven central soon. i had some issues with the sonatype flow so had to do a manual release. if it doesn't work, let me know and i'll try and repair my sonatype flow.

**Comment by christobill** [comment]
@mpollmeier v2.6.2 is in maven central. Thank you for the quick update!
